index.bs

```diff
@@ -2216,6 +2218,47 @@ The primary difference between {{XRViewerPose}} and {{XRPose}} is the inclusion
 
 If the relationship between {{XRView}}s could uniquely identify the [=/XR device=], then the user agent MUST anonymized the {{XRView}} data to prevent fingerprinting. The method of anonymization is at the discretion of the user agent. Furthermore, if the relationship between {{XRView}}s is affected by a user-configured interpupillary distance, then it is strongly recommended that the user agent require [=explicit consent=] prior to reporting any {{XRView}} data.
 
+### Reference spaces ### {#protect-reference-spaces}
+Various reference space types have restrictions places on their use to prevent the unintentional exposure of [=sensitive information=].
+
+For any {{XRReferenceSpaceType}} included in an {{XRSession}}'s [=requested features=] the user agent MUST ensure the document is allowed to use all policy-controlled features associated with the sensor types used to track the [=native origin=] of the {{XRReferenceSpaceType}} or the feature MUST be rejected.
```

Comment:
Per comment in #761 I wonder if this text needs to be adjusted slightly:

```suggestion
For any {{XRReferenceSpaceType}} included in an {{XRSession}}'s [=requested features=] the user agent MUST ensure the document is not explicitly blocked from using any of the policy-controlled features associated with the sensor types used to track the [=native origin=] of the {{XRReferenceSpaceType}}, or the feature MUST be rejected.
```

@ddorwin WDYT?